umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2.clj

```diff
@@ -75,6 +75,10 @@
   (str "/gmi:MI_Metadata/gmd:metadataExtensionInfo/gmd:MD_MetadataExtensionInformation"
        "/gmd:extendedElementInformation/gmd:MD_ExtendedElementInformation"))
 
+(def collection-data-type-xpath
+  (str identifier-base-xpath "[gmd:codeSpace/gco:CharacterString='gov.nasa.esdis.umm.collectiondatatype']"
```

Comment:
The generated tests produce collection data types as well as the example_data/iso19115/ISOExample-project.xml